applications/GeoMechanicsApplication/tests/lysmer_boundary_column2d_quad/README.md

```diff
@@ -0,0 +1,34 @@
+# Lysmer absorbing boundary conditions on a column made of rectangulars
+
+This test verifies the implementation of the Lysmer absorbing boundary conditions. 
+
+## Setup
+
+The test is performed for an 1D column that is made with 2D rectangular elements. The column height and width are 10,000 mm and 1,000 m, respectively.
+The column consists of ten 4-noded elements (using the UPwSmallStrainElement2D4N class) of 1,000mm x 1,000 mm. A
+schematic representation can be found in the figure below:
+
+![MeshStructure](MeshStructure.svg)
+
+All Nodes have fixed displacement in the X direction. 
+
+An instant load of -10 N is applied in the Y direction at the column top that is defined by Nodes 1 and 2. 
+
+The absorbing boundary conditions are applied on a line created by Nodes 21 and 22. The unity absorbing factors are used there.
+
+The material is described using:
+
+-   A linear elastic plane stress model (LinearElasticPlaneStrain2DLaw),
+-   Young's modulus is 10,000 N/mm2
+-   Poisson's ratio is 0.2,
+-   A thickness of 1 mm,
+-   Density of solid is 2.65 kg/mm3,
```

Comment:
2.65 for soil density makes me think that the unit of mass is ton, the unit of length is m and the corresponding unit of force is kN.

Reply:
Changed kg => ton and put kN/m2 for the Young's module. Hope the value is consistent with data for sand.